recipes/quake/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "quake" %}
+{% set version = "0.3.5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+about:
+  home: 'http://www.cbcb.umd.edu/software/quake/'
+  summary: "Quake is a package to correct substitution sequencing errors in experiments with deep coverage (e.g. >15X), specifically intended for Illumina sequencing reads"
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://www.cbcb.umd.edu/software/quake/downloads/{{ name }}-{{ version }}.tar.gz
+  md5: f8e26c0c29d22fc0d49fcfb5dc8945d3
+
+build:
+  number: 0
+  skip: True # [py3k]
```

Comment:
If you pin to boost, you need to change the build string, similar to: https://github.com/bioconda/bioconda-recipes/blob/19d529db7da011a5225d35f3e984021d0063b7c9/recipes/sailfish/meta.yaml#L4